examples/investigate_streams.rs

```diff
@@ -242,21 +262,23 @@ fn main() {
             lzma: "lzma (b)",
         },
     );
-    let print = |in_extension: BTreeMap<String, Compressions<u64>>| {
-        for (name, info) in in_extension {
-            print_column(&options, &name, &info);
-        }
-    };
 
     // Output `js` and `binjs` first.
-    if let Some(in_extension) = stats.remove("js") {
-        print(in_extension);
+    if let Some(mut in_extension) = stats.remove("js") {
+        print_column(&options, "js", &in_extension.remove("js").unwrap());
     }
-    if let Some(in_extension) = stats.remove("binjs") {
-        print(in_extension);
+    if let Some(mut in_extension) = stats.remove("binjs") {
+        print_column(&options, "binjs", &in_extension.remove("binjs").unwrap());
     }
-    // Output the rest by alphabetical order.
-    for (_, in_extension) in stats {
-        print(in_extension);
+    println!("");
+
+    // Output the rest by `ranking` order.
+    let all_values = stats.into_iter().flat_map(|(_, value)| value.into_iter());
+    let sorted = match ranking {
+        Ranking::Alpha => all_values.sorted(),
+        Ranking::BrotliCompressedSize => all_values.sorted_by_key(|x| x.1.brotli),
```

Comment:
Hmm maybe just collect into `Vec` and then do `sort_by_key`? Then you don't need itertools as a dependency here at all.